common/authorization/frontend_api.go

```diff
@@ -37,7 +37,6 @@ var readOnlyNamespaceAPI = map[string]struct{}{
 	"QueryWorkflow":                      {},
 	"DescribeWorkflowExecution":          {},
 	"DescribeTaskQueue":                  {},
-	"GetSystemInfo":                      {},
```

Comment:
Should keep it here.

Reply:
how about we move it to readOnlyGlobalAPI? it's not a namespace api at all